### ISSUE TITLE
Remove warnings in React code-sandbox

### ISSUE THREAD

**Issue by curquiza** [meilisearch/instant-meilisearch#30]
Currently, we get these warnings in the console in [React codesandbox](https://codesandbox.io/s/ms-react-is-sh9ud?file=/src/App.js):

```bash
Warning: Failed prop type: The prop `hits[0].objectID` is marked as required in `Hits`, but its value is `undefined`.
```

```bash
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Hits`. See https://fb.me/react-warning-keys for more information.
    in li (created by Hits)
    in Hits (created by AlgoliaHits(Hits))
    in AlgoliaHits(Hits) (created by Context.Consumer)
    in ConnectorWrapper (at App.js:55)
    in div (at App.js:53)
    in InstantSearch (at App.js:37)
    in div (at App.js:25)
    in App (at src/index.js:8)
    in StrictMode (at src/index.js:7)
```

**Comment by curquiza** [comment]
Hello @iamanishagarwal, I assign you! 🙂

**Comment by iamanishagarwal** [comment]
Hello @curquiza , I am facing this issue while running the playground react in development mode.
![Capture](https://user-images.githubusercontent.com/30866925/95108020-4faa2800-0758-11eb-84cd-b45e055d000b.PNG)

**Comment by curquiza** [comment]
Hello @iamanishagarwal! Looks weird, I don't have this error at all:

```
$ git clone git@github.com:meilisearch/instant-meilisearch.git
[...]
$ cd instant-meilisearch
$ yarn
[...]
$ yarn playground:react
```

<img width="409" alt="Capture d’écran 2020-10-06 à 11 02 44" src="https://user-images.githubusercontent.com/20380692/95181462-9da84580-07c3-11eb-87ba-818444c17c7b.png">

**Comment by bidoubiwa** [comment]
Hey! This has been fixed with #322 
You can now pass the primaryKey as an option 
```javascript
const searchClient = instantMeiliSearch(
  'https://demos.meilisearch.com',
  'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25',
  {
    paginationTotalHits: 60,
    primaryKey: 'id',
  }
)
```
And i will use that field of your documents to generate the `objectID`

**Comment by Wamy-Dev** [comment]
> Hey! This has been fixed with #322 You can now pass the primaryKey as an option
> 
> ```js
> const searchClient = instantMeiliSearch(
>   'https://demos.meilisearch.com',
>   'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25',
>   {
>     paginationTotalHits: 60,
>     primaryKey: 'id',
>   }
> )
> ```
> 
> And i will use that field of your documents to generate the `objectID`

Thank you this works!
